viogpu/viogpudo/viogpudo.cpp

```diff
@@ -600,7 +600,7 @@ NTSTATUS VioGpuDod::QueryVidPnHWCapability(_Inout_ DXGKARG_QUERYVIDPNHWCAPABILIT
     pVidPnHWCaps->VidPnHWCaps.DriverScaling = 0;
     pVidPnHWCaps->VidPnHWCaps.DriverCloning = 0;
     pVidPnHWCaps->VidPnHWCaps.DriverColorConvert = 1;
-    pVidPnHWCaps->VidPnHWCaps.DriverLinkedAdapaterOutput = 0;
+    pVidPnHWCaps->VidPnHWCaps.DriverLinkedAdapterOutput = 0;
```

Comment:
'DriverLinkedAdapaterOutput' is MS API (https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/d3dkmdt/ns-d3dkmdt-_d3dkmdt_vidpn_hw_capability) . Please remove this change.